handbook/customer-success/README.md

```diff
@@ -0,0 +1,226 @@
+# Customer Success
+
+This handbook page details processes specific to working [with](#contact-us) and [within](#responsibilities) this department.
+
+## What we do
+The customer success department is directly responsible for ensuring that customers and community members of Fleet achieve their desired outcomes with Fleet products and services.
+
+
+## Team
+| Role                                  | Contributor(s)           |
+|:--------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
+| VP of Customer Success                | [Zay Hanlon](https://www.linkedin.com/in/zayhanlon/) _([@zayhanlon](https://github.com/zayhanlon))_
+| Customer Success Manager (CSM)        | [Jason Lewis](https://www.linkedin.com/in/jlewis0451/) _([@patagonia121](https://github.com/patagonia121))_
+| Customer Support Engineer (CSE)       | [Kathy Satterlee](https://www.linkedin.com/in/ksatter/) _([@ksatter](https://github.com/ksatter))_, [Grant Bilstad](https://www.linkedin.com/in/grantbilstad/) _([@Pacamaster](https://github.com/Pacamaster))_, Ben Edwards _([@edwardsb](https://github.com/edwardsb))_
+| Infrastructure Engineer               | [Robert Fairburn](https://www.linkedin.com/in/robert-fairburn/) _([@rfairburn](https://github.com/rfairburn))_
+
+## Contact us
+- To make a request of this department, [create an issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-customer-success&projects=&template=custom-request.md&title=Request%3A+_______________________) and a team member will get back to you within one business day.
+  - Any Fleet team member can [view the kanban board](https://github.com/fleetdm/fleet#workspaces/-g-customer-success-642c83a53e96760014c978bd/board) for this department, including pending tasks and the status of new requests.
+  - Please use issue comments and GitHub mentions to communicate follow-ups or answer questions related to your request.
+- If urgent, or if you need help submiting your request, mention a [team member](#team) in the [#g-customer-success](https://fleetdm.slack.com/archives/C062D0THVV1) Slack channel.
+
+
+## Customer support
```

Comment:
@zayhanlon @ksatter @rfairburn @Patagonia121 @pacamaster   I know right now we're doing a lift and shift, but would love to see this philosophy stuff smooshed into responsibility entries if possible.  @Sampfluger88 can explain

Reply:
(when you can.  I'ma go ahead and merge it after I fix whatever is wrong with the lint)